website/docs/ui/assistant-ui.md

```
@@ -197,6 +213,9 @@ export const Assistant = () => {
 
 ```bash
 pnpm dev
+pnpm voltagent:run
 ```
 
 Open http://localhost:3000 and chat—the UI will stream reasoning and messages from your VoltAgent agent. Attachments and thread persistence are handled automatically via the VoltAgent memory adapter.
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)